### ISSUE TITLE
Jupyter Notebook based OpenMined Tutorials

### ISSUE THREAD

**Issue by iamtrask** [OpenMined/PySyft#322]
This is a longstanding ticket (DO NOT CLOSE!)

**Description:** Whenever we build a new piece of functionality, the most important factor is whether or not the rest of the community can quickly and easily understand and begin to use the feature. To that end, we would like to think of every possible use of the OpenMined platform and build Notebooks which give tutorials for this functionality. Furthermore, we want to take EXISTING notebooks and make them easier for people to use by adding better text descriptions... step by step instructions... etc. Here are a few [examples](https://github.com/OpenMined/PySyft/tree/master/notebooks) for your inspiration.

Particularly if you are a beginner, this is the issue for you.

**Acceptance Criteria**
- Find an existing notebook and make it easier to run / understand
OR
- Write a new notebook demonstrating a new potential use of the OpenMined platform (or a variant on an existing one). Upload your new notebook into the following folder "PySyft/notebooks".

**Examples:**
- Train one of our PySyft models on a new dataset!
- Combine multiple different tensor types to build a classifier!
- Compare and contrast the speed of different encryption algorithms in the same classifier.
- _insert your idea here_

**Comment by chunduri11** [comment]
So this issue is about revamping the available notebooks for easy use, and creating new notebooks to use PySyft base functions.
After this people should not find it difficult to setup dependencies and should be able to test their dependencies easily.

Is this a single issue or multiple issues, depending on number of notebooks to be edited and created.

**Comment by iamtrask** [comment]
Single issue but multiple PRs :)

**Comment by vasu-dev** [comment]
i would like to train the PySyft model on mnist-fashion if not already done.

**Comment by vasu-dev** [comment]
I would like to train PySyft model on mnist-fashion dataset , if not already done.

**Comment by planetceres** [comment]
I'd like to work on this. Do I need to choose a specific tutorial topics to this issue, or just submit pull requests once I have them completed?

**Comment by iamtrask** [comment]
@planetceres just PRs when they're completed is fine :)

**Comment by dineshkumaramara1** [comment]
I would like to write a new simplified code and commit the notebook

**Comment by nahuakang** [comment]
Hi @vasu-dev @planetceres @dineshkumaramara1! Just wondering what y'all are working on (I saw one MNIST)? I'd also like to contribute here and would like to communicate with you on Slack before starting.

**Comment by bharathgs** [comment]
@vasu-dev @planetceres @nahuakang @dineshkumaramara1 @alberduris it seems like you folks are interested in this issue, if that is the case, I would also add that you can take a look at [this](https://github.com/OpenMined/tutorials/issues/2).
